client/signup/config/flows-pure.js

```diff
@@ -397,6 +397,16 @@ export function generateFlows( {
 			showRecaptcha: true,
 			hideProgressIndicator: true,
 		},
+		{
+			name: 'domain-for-gravatar',
+			steps: [ 'domain-only', 'site-or-domain', 'site-picker' ],
+			destination: getDomainSignupFlowDestination, // TODO: Redirect to Gravatar
```

Comment:
We might want to either create a new thank you page for the Gravatar flow or somehow redirect to the Gravatar profile page when checkout completes